content/en/database_monitoring/data_collected.md

```diff
@@ -22,9 +22,39 @@ The metrics used for Database Monitoring views are, primarily:
 - **MySQL**: `mysql.queries.*`
 - **Postrgres**: `postgresql.queries.*`
 
+### Normalized queries ###
+
+In order to eliminate redundant information and keep track of performance trends, identical queries with different parameters are grouped together by obfuscating the parameters. These query groups are called normalized queries and sometimes referred to as query digests. Rather than imposing a strict query volume limitation, Datadog supports 200 normalized queries per database host. This also ensures that no sensitive data leaks into observability tools.
+
+For example, you may see many queries retrieving data from the same table by id:
+
+```sql
+SELECT * FROM customers WHERE id = 13345;
+SELECT * FROM customers WHERE id = 24435;
+SELECT * FROM customers WHERE id = 34322;
+```
+
+These will all appear as a normalized query that replaces parameters with ?
+
+```sql
+SELECT * FROM customers WHERE id = ?
+```
+
+Any number of parameters will follow the same pattern:
+
+```sql
+SELECT * FROM timeperiods WHERE start >= '2022-01-01' AND end <= '2022-12-31' AND num = 5
+```
+
+becomes
+
+```sql
+SELECT * FROM timeperiods WHERE start >= ? AND end <= ? AND num = ?
+```
+
 ## Sensitive information
 
-The Database Monitoring Agent obfuscates all query bind parameters sent to the Datadog intake. Thus passwords, PII (Personally identifiable information), and other potentially sensitive information stored in your database will not be viewable in query metrics, query samples, or explain plans.
+Because Database Monitoring Agent normalizes queries, it obfuscates all query bind parameters sent to the Datadog intake. Thus passwords, PII (Personally identifiable information), and other potentially sensitive information stored in your database will not be viewable in query metrics, query samples, or explain plans.
```

Comment:
```suggestion
Because Database Monitoring Agent normalizes queries, it obfuscates all query bind parameters sent to the Datadog intake. Thus, passwords, PII (Personally identifiable information), and other potentially sensitive information stored in your database are not viewable in query metrics, query samples, or explain plans.
```
I know you didn't originally write the words I'm complaining about, but let's fix them up while we're here.